elements/pfelement/src/pfelement.js

```diff
@@ -1,11 +1,31 @@
 import { autoReveal } from "./reveal.js";
-const prefix = "pfe-";
+import { isAllowedType, isValidDefaultType } from "./attrDefValidators.js";
+
+const tagPrefix = "pfe-";
```

Comment:
```suggestion
const prefix = "pfe-";
```

I think prefix is a good name here because it's still the prefix for tags and attributes; we're just additionally adding scope information to make it `${prefix}-${scope}-${name}` 🤷